.travis.yml

```diff
@@ -1,85 +1,115 @@
 # https://docs.travis-ci.com/user/customizing-the-build/
 language: cpp
-
-sudo: required
-
+cache: ccache
 matrix:
   include:
     - os: linux
+      dist: trusty
+      compiler: gcc
+      env: DEFAULT_TRAVIS_COMPILER="true" TRAVIS_DIST="trusty"
```

Comment:
The default uses gcc-4. Do we want to keep a test for that or just test gcc-5-8?

Reply:
Reviewing GCC default versions currently in *supported* [Debian](https://qa.debian.org/madison.php?package=gcc&table=debian) and [Ubuntu](https://qa.debian.org/madison.php?package=gcc&table=ubuntu) releases, I think the oldest gcc version MegaGlest should currently try to support is Ubuntu 16.04 (LTS) "Xenial"'s gcc 5.3.1.